exciting_environments/cart_pole/cart_pole_env.py

```diff
@@ -125,7 +126,7 @@ class PhysicalState:
         omega: jax.Array
 
     @jdc.pytree_dataclass
-    class Optional:
+    class Additions:
         """Dataclass containing additional information for simulation."""
 
         something: jax.Array
```

Comment:
It looks kinda janky to have this, with `something` if it is set to `None` anyways